tests/posix/common/src/pthread.c

```diff
@@ -371,6 +377,67 @@ ZTEST(pthread, test_pthread_termination)
 	zassert_equal(ret, ESRCH, "cancelled a terminated thread!");
 }
 
+ZTEST(pthread, test_pthread_tryjoin)
+{
+	pthread_t th = {0};
+	void *retval;
+
+	/* Creating a threads that exits after 1s*/
```

Comment:
The comments should match the updated sleep time

Reply:
adjusted. Nice catch.